public/app/features/variables/adhoc/picker/ConditionSegment.tsx

```diff
@@ -6,7 +6,7 @@ interface Props {
 
 export const ConditionSegment: FC<Props> = ({ label }) => {
   return (
-    <div className="gf-form">
+    <div className="gf-form" data-wtf="condition-segment">
```

Comment:
Just out of curiosity, what's this data attribute?

Reply:
oop my bad i should read my diffs before submitting them more 🙈